rpmbuild/wacom-gui.spec

```diff
@@ -1,32 +1,31 @@
-%global libwacom_ver 1.11
+%global libwacom_ver 2.2.0
```

Comment:
On EL8 I currently get a `libwacom-1.6-3.el8.x86_64` - don't know whatever the version is important here?